bindings/python/py_src/tokenizers/trainers/__init__.pyi

```diff
@@ -7,19 +7,6 @@ class Trainer:
     Trainer will return an instance of this class when instantiated.
     """
 
-    def __init__(
-        self,
-        vocab_size=30000,
-        min_frequency=0,
-        show_progress=True,
-        special_tokens=[],
-        limit_alphabet=None,
-        initial_alphabet=[],
-        continuing_subword_prefix=None,
-        end_of_word_suffix=None,
-    ):
-        pass
-
```

Comment:
these lines were removed because of[ this change](https://github.com/huggingface/tokenizers/pull/949#discussion_r823826088) ?